build-system/tasks/serve.js

```diff
@@ -96,8 +94,6 @@ gulp.task(
         'cache': '  Make local resources cacheable by the browser ' +
             '(default: false)',
         'inspect': '  Run nodemon in `inspect` mode',
-        'enable_dev_dashboard_cache':
-            '  Enables the dev dashboard cache (default: false)',
```

Comment:
Remove this flag from `gulpfile.js` too

https://github.com/ampproject/amphtml/blob/1c54451b82cb9048b566d33ea3ad99e3a820183f/gulpfile.js#L1699-L1700